server/models/k8s_context.go

```diff
@@ -224,14 +229,15 @@ func NewK8sContext(
 
 	ID, err := K8sContextGenerateID(ctx)
 	if err != nil {
-		return ctx
+		return ctx, ""
 	}
 
 	ctx.ID = ID
+	msg := fmt.Sprintf("Generated context: %s\n", ctx.Name)
 
-	logrus.Infof("Generated context: %s\n", ctx.Name)
+	logrus.Infof(msg)
 
-	return ctx
+	return ctx, msg
```

Comment:
Please send the `contexts` name list in response. UI can present the message accordingly.